cypress/support/TestListManager.js

```diff
@@ -125,18 +125,9 @@ class TestListIterator {
 }
 
 function GetTestGroup(name) {
```

Comment:
This is a bit misleading. Would prefer to name the method in a way that indicates how it will behave. `findTestGroup`
It is named `Get` but internally uses `find`. These have different meanings.

To me `get` prefixed methods are like a getter. It should be guaranteed to get the result of the function. Otherwise something is wrong and it is an error. In which case it should `throw`

`find` prefixed method imply the are searching or trying to find something. It is not guaranteed they will succeed and in the case they don't find it they return `undefined`

Even the language authors chose `find` instead of `get` for this type of method.  I think we should preserve their naming to avoid confusion.

Reply:
I made a note to self to review this code and fix it, but later not for this PR.